### PR TITLE
Added dtumathtools

### PR DESCRIPTION
**Pull request by zerothi**
This package is very simple, it basically wraps the requirements files for the students at DTU for a specific course.

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by zerothi** [comment]
@Chrillebon, could you also acknowledge here that you agreed to be a maintainer?

**Comment by Chrillebon** [comment]
I agree to be maintainer!

In terms of the plotting tests, this is another example of why we might need to transition to [tests not requiring all backends](https://github.com/dtudk/dtumathtools/issues/13), and should thus not be fixed here.

One comment - I think we should add python 3.12 to the tests being run, if we want to support 3.12 going forward. That is, in the future, we should add 3.12 into the tests.

Otherwise, it is looking good.

**Comment by nicpa-dtu** [comment]
accepted maintainer

**Comment by zerothi** [comment]
@conda-forge/help-python ready for review

File: recipe/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: f9fe7e34dc5be3954f451ea639c3ab47ba31c0b2b620f5e29618f1640bce1782
+
+build:
```
Comment (synapticarbors) [inline recipe/meta.yaml:12]: Can this package be `noarch: python`? I don't see any compiled pieces. 

https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python

**Review by synapticarbors — CHANGES_REQUESTED** [review]
Your PR should create a directory inside `recipes` (so `recipes/dtumathtools`) and not touch any files outside of that. Please refactor the PR to adhere to that.

File: recipe/meta.yaml (hunk 1 above)
Comment (zerothi) [reply recipe/meta.yaml]: thanks a lot for your comments, sorry about the directory stuff.

Indeed the `noarch: python` may be applied here. I have added that, and removed the `requirements.build` for the `mac` stuff... Fingers cross that it works.

**Pull request merged**